### PR TITLE
Updating binding messaging for consistency

### PR DESCRIPTION
**Pull request by rhamilto**
and changing bind-specific CSS rules to be more generic so they can be reused for provisioning in catalog.

This is the first of three PRs to update messaging across catalog/console.  I'll need a release of common once this merges to make use of in catalog and console.

screen shot 2017-09-26 at 11 21 59 am
screen shot 2017-09-26 at 11 22 11 am

### CODE REVIEW

File: src/components/binding/bindResults.js [hunk 1]
```diff
@@ -15,16 +14,5 @@ angular.module('openshiftCommonUI').component('bindResults', {
   templateUrl: 'src/components/binding/bindResults.html',
   controller: function() {
```
Comment (spadgett) [inline src/components/binding/bindResults.js]: You should be able to remove `controller` entirely now.

File: src/components/binding/bindResults.html [hunk 2]
```diff
@@ -1,19 +1,17 @@
...
-    <div class="bind-status">
+    <div class="results-status">
       <span class="pficon pficon-ok" aria-hidden="true"></span>
       <span class="sr-only">Success</span>
-      <h3 class="bind-message">
-        <strong>{{ctrl.serviceToBind}}</strong>
-        <span>has been bound</span>
-        <span ng-if="ctrl.bindType === 'application'"> to <strong>{{ctrl.applicationToBind}}</strong> successfully</span>
+      <h3 class="results-message">
+        <strong>{{ctrl.serviceToBind}}</strong> has been bound <span ng-if="ctrl.bindType === 'application'"> to <strong>{{ctrl.applicationToBind}}</strong></span> successfully.
```
[the remark this answers is not in the document]
Comment (ncameronbritt) [reply src/components/binding/bindResults.html]: That makes sense to me.

**Comment by ncameronbritt** [comment]
I think we should be using the service class display name rather than the service instance name since that's what we show most prominently on the Overview.

File: src/components/binding/bindResults.html [hunk 3]
```diff
@@ -23,19 +21,19 @@ <h3 class="bind-message">
...
 </div>
 <div ng-if="ctrl.error || (ctrl.binding | isBindingFailed)">
-  <div class="bind-status">
+  <div class="results-status">
     <span class="pficon pficon-error-circle-o text-danger" aria-hidden="true"></span>
     <span class="sr-only">Error</span>
-    <h3 class="bind-message">
-      <span>Binding Failed</span>
+    <h3 class="results-message">
+      The binding operation failed.
```
[the remark this answers is not in the document]
Comment (ncameronbritt) [reply src/components/binding/bindResults.html]: I agree with that.

File: src/components/binding/bindResults.html (hunk 2 above)
[the remark this answers is not in the document]
Comment (ncameronbritt) [reply src/components/binding/bindResults.html]: Is something like "Binding created successfully" sufficient for this context? Not sure it's necessary to name the service again.

[the remark this answers is not in the document]
Comment (spadgett) [reply src/components/binding/bindResults.html]: "Binding created successfully." works for me

File: src/components/binding/bindResults.html [hunk 4]
```diff
@@ -23,19 +21,19 @@ <h3 class="bind-message">
       that you may need to reference in your application.
       <span ng-if="ctrl.showPodPresets">Its data will be available to your application as environment variables.</span>
```
Comment (rhamilto) [inline src/components/binding/bindResults.html]: @spadgett, should I remove this, too, since pod presets are out?

**Comment by rhamilto** [comment]
@spadgett, comments addressed.

**Pull request merged**